hadoop-hdfs-project/hadoop-hdfs-nfs/src/test/java/org/apache/hadoop/hdfs/nfs/nfs3/TestDFSClientCache.java

```diff
@@ -31,8 +31,14 @@
 import org.apache.hadoop.hdfs.nfs.conf.NfsConfiguration;
 import org.apache.hadoop.security.UserGroupInformation;
 import org.junit.Test;
+import org.junit.After;
```

Comment:
import order is wrong, it needs to be sorted alphabetically, put .After above .Test
```suggestion
import org.junit.After;
import org.junit.Test;
```

Reply:
Updated. Thanks for the reminder.